packages/manager/apps/telecom/src/app/telecom/pack/dashboard/pack.module.js

```diff
@@ -1,12 +1,32 @@
 import angular from 'angular';
 
 import packMove from '../move';
+import packResiliation from '../resiliation/pack-resiliation';
 import packVoipLineActivation from '../slots/voipLine/activation/pack-voipLine-activation.module';
+import packEcoFaxActivation from '../slots/voipEcoFax/activation';
```

Comment:
Is it possible to have a `slots` module in `../slots` that import children modules  ?

Here all imports could be replaced by `import slots from '../slots';`.